grafana-charts/README.md

```diff
@@ -2,19 +2,234 @@
 
 ### Prerequisites
 * OpenEBS installed
-* OpenEBS Volumes created
+* OpenEBS Volumes created(up and running)
 
-### Steps
+### Steps 
+* When applying the configs using [openebs-monitoring-pg.yaml](https://raw.githubusercontent.com/Ab-hishek/openebs-monitoring/master/openebs-monitoring-pg.yaml):
 1. Run kubectl apply -f [openebs-monitoring-pg.yaml](https://raw.githubusercontent.com/Ab-hishek/openebs-monitoring/master/openebs-monitoring-pg.yaml)
-2. Run `kubectl get pods -o wide` to verify and to get where Prometheus and Grafana instances are running.
+2. Run `kubectl get pods -n openebs -o wide` to verify and to get where Prometheus and Grafana instances are running.
 3. Run `kubectl get nodes -o wide` to get the node’s IP.
-4. Run `kubectl get svc` and note down the port allocated to Prometheus and Grafana services.
+4. Run `kubectl get svc -n openebs` and note down the port allocated to Prometheus and Grafana services.
 5. Open your browser and open `<nodeip:nodeport>`(32514 for Prometheus and 32515 for Grafana) and you should be able to see the Prometheus’s expression browser and Grafana’s UI on your browser.
-6. Login into it(default username and password is admin), add data source name as prometheus, select datasource type Prometheus and pass the `<nodeip:nodeport>`of Prometheus in the url field and click on run and test.
+6. Login into it(default username and password is admin), add data source name as DS_OPENEBS_PROMETHEUS, select datasource type Prometheus and pass the `<nodeip:nodeport>`of Prometheus in the url field and click on run and test.
 7. Now go to the next tab (Dashboard) and import the desired dashboard.You should be able to get preloaded dashboard of prometheus.
 8. To create openebs dashboard, go to the import and paste the json from this folder.
```

Comment:
To import OpenEBS dashboards, you will to import them separatly (please..fix my english. and typos.) You can upload the json files one by one, or open a json file and copy paste the content.   Also..specify where are the dashaobard files.. like 
clustervolumesdashboard.json  

![image](https://user-images.githubusercontent.com/191879/110807708-52ea9800-8251-11eb-8f74-00e7c9522ebd.png)

select the datasource click import

![image](https://user-images.githubusercontent.com/191879/110807798-64cc3b00-8251-11eb-8f47-ef54abe5f4c0.png)

repeat the process for all dashboard files